### ISSUE TITLE
Left nav lacks visible tab or strong focus indicators with Windows HC modes

### ISSUE THREAD

**Issue by DaveTryon** [microsoft/accessibility-insights-web#4263]
## Describe the bug

If you enable a Windows HC mode, the left nav pane loses both the hover indicator and the current selection indicators

**To Reproduce**

1. Start an assessment on a page (I used )
2. Hover over the assessment steps in the nav pane -- there's no strong visual feedback (except the tooltip if you pause long enough)
3. Select an item in the list--there should be a clear indicator that it's selected, but the only indication seems to be a bolding of text (which is much harder to discern than the non-HC version)

Alternatively, run FastPass--all 3 steps in the nav bar look more or less the same

## Expected behavior

There should be clearly visible counterparts of the selection in HC modes

## Screenshots

In these screenshots, the mouse hover is on "3 Focus", and the selection is on "3.1 Visible Focus". The only The Black-based HC themes show a "thumb"

HC Mode | Screenshot
--- | ---
None | ![image](https://user-images.githubusercontent.com/45672944/119208440-0c869600-ba57-11eb-87ba-931929026d94.png)
HC 1 | ![image](https://user-images.githubusercontent.com/45672944/119207236-6173dd80-ba52-11eb-8bfd-555764ca658d.png)
HC 2 | ![image](https://user-images.githubusercontent.com/45672944/119207104-07731800-ba52-11eb-9613-1602adf67ed0.png)
HC Black | ![image](https://user-images.githubusercontent.com/45672944/119208353-aef24980-ba56-11eb-8219-e50ad6c76b00.png)
HC White | ![image](https://user-images.githubusercontent.com/45672944/119208396-d6e1ad00-ba56-11eb-9a3a-30bc48d9b488.png)

## Context (please complete the following information)

- OS Name & Version: Windows 10 21H1
- AI-Web Version & Environment: 2.26.0 Production
- Browser Version: Chrome Version 90.0.4430.212 (Official Build) (64-bit)
- Target Page: https://www.washington.edu/accesscomputing/AU/before.html

**Comment by ferBonnin** [comment]
@ferBonnin needs some PM/ Design information

**Comment by ferBonnin** [comment]
@jlperkins could you please help us verify if this issue still repos after the Fluent update?

**Comment by jlperkins** [comment]
With the latest changes in main, and the new HC modes in Windows 11, this issue still repros.  As seen in the images below, the assessment step gets a small colored bar to the side, and the individual assessment task in question gets bolded, but there is no strong indication that you are on that step or have selected that task. 

![image](https://user-images.githubusercontent.com/33770444/165598634-7c719dd8-7946-4c3e-a64c-b71c6c477a05.png)
[Theme 1, Aquatic]

![image](https://user-images.githubusercontent.com/33770444/165598671-67e8688d-8555-4c8f-a01f-93b2a6f20d63.png)
[Theme 2, Desert]

![image](https://user-images.githubusercontent.com/33770444/165598693-df4e18dc-22bb-4d94-9a62-b349b464299a.png)
[Theme 3, Dusk]

![image](https://user-images.githubusercontent.com/33770444/165598767-a6991363-47dc-4e7f-a2d7-f87aa0af333f.png)
[Theme 4, Night Sky]

**Comment by v-viyada** [comment]
@nang4ally 
Please review below screenshots if they are the desired outcomes for this bug or any changes required in this.

In these screenshots, the mouse hover is on "3 Focus", and the selection is on "3.1 Visible Focus"
[Theme 1 Aquatic]
<img width="234" alt="image" src="https://github.com/microsoft/accessibility-insights-web/assets/95463767/2047e235-7472-4d2b-8552-c4a141996426">

[Theme 2 Desert]
<img width="231" alt="image" src="https://github.com/microsoft/accessibility-insights-web/assets/95463767/6ea575ec-efcb-4750-8751-0efe1ce2ecb1">

[Theme 3 Dusk]
<img width="227" alt="image" src="https://github.com/microsoft/accessibility-insights-web/assets/95463767/27307588-e4c0-4b63-b1a2-e1afff319e5d">

[Theme 4 Night Sky]
<img width="228" alt="image" src="https://github.com/microsoft/accessibility-insights-web/assets/95463767/af6a8b3e-6126-4572-8b62-c57d553bf3b7">

@peterdur FYI
